lib/maze/logger.rb

```diff
@@ -17,7 +17,12 @@ def initialize
       else
         super(STDOUT, level: Logger::INFO)
       end
-      self.datetime_format = '%Y-%m-%d %H:%M:%S'
+
+      self.formatter = proc do |severity, time, _name, message|
+        formatted_time = time.strftime('%H:%M:%S')
```

Comment:
```suggestion
        formatted_time = time.strftime(ENV['MAZE_LOG_FORMAT'] || '%H:%M:%S')
```
I like the change, but could we factor the string format out into `ENV['MAZE_LOG_FORMAT']` to stave off any complaints about losing the date?  

If you agree, can you put a short note for it in the Logging section of `DOCS.md`?

Reply:
I've done this though using the [`Logger#datetime_format` attribute](https://rubyapi.org/3.1/o/logger#method-i-datetime_format) instead of an environment variable

I've also added docs for overwriting the formatter completely, which is just the [default `Logger#formatter` behaviour](https://rubyapi.org/3.1/o/logger#formatter)